docs/signer.md

```diff
@@ -0,0 +1,354 @@
+If enabled, any image a user pushes to Google Container Registry, or gcr.io, will be automatically scanned for vulnerability. 
+
+#### Vulnerability Signing Policy
+
+A vulnerability signing policy yaml file can be specified via `-policy` flag, and controls requirements 
+for an image to pass vulnerability-based checks.
+
+#### Vulnerability Signing Policy Spec
+
+| Field     | Default (if applicable)   | Description |
+|-----------|---------------------------|-------------|
+|packageVulnerabilityPolicy.maximumFixableSeverity| CRITICAL | Tolerance level for vulnerabilities found in the container image.|
+|packageVulnerabilityPolicy.maximumUnfixableSeverity |  ALLOW_ALL | The tolerance level for vulnerabilities found that have no fix available.|
+|imageVulnerabilityRequirements.allowlistCVEs |  | List of CVEs which will be ignored, has to be full note name.|
+
+Example policy:
+
+```
+apiVersion: kritis.grafeas.io/v1beta1
+kind: VulnzSigningPolicy
+metadata:
+  name: my-vsp
+spec:
+  imageVulnerabilityRequirements:
+    maximumFixableSeverity: MEDIUM
+    maximumUnfixableSeverity: MEDIUM
+    allowlistCVEs:
+      - projects/goog-vulnz/notes/CVE-2020-10543
+      - projects/goog-vulnz/notes/CVE-2020-10878
+      - projects/goog-vulnz/notes/CVE-2020-14155
+```
+
+Here are the valid values for severity levels.
+
+| Value       | Outcome |
+|-------------|----------- |
+| BLOCK_ALL | Block all vulnerabilities except listed in allowlist. |
+| LOW | Allow Containers with Low  vulnerabilities. |
+| MEDIUM | Allow Containers with Low and Medium vulnerabilities. |
+| HIGH  | Allow Containers with Low, Medium & High vulnerabilities. |
+| CRITICAL  | Allow Containers with all known vulnerability levels (LOW to CRITICAL). |
+| ALLOW_ALL | Allow all vulnerabilities, including vulnerability with unknown severity levels. |
+
+### Signing
+
+Signing is the process of creating and storing an attestation for an image.
+To create an attestation, a private signing key is needed. The signer tool now
+supports three types of signing methods: PGP keys, PKIX keys, or signing via [Cloud KMS] (https://cloud.google.com/kms).
+
+By default, the signing helper will also upload the attestation occurrence to [Google Container Analysis](https://cloud.google.com/container-registry/docs/container-analysis),
+and the uploaded attestation can be used by both Binary Authorization and Kritis for enforcement-time decisions.
+
+#### Supported key types
+
+The tool currently supports three types of keys:
+
+- PGP keys using `-pgp_private_key` and optionally `-pgp_passphrase`  if the key is passphrase protected.
+- PKIX keys using `-pkix_private_key` and `-pkix_alg`. Supported PKIX algorithms match those [supported by Binary Authorization](https://cloud.google.com/sdk/gcloud/reference/container/binauthz/attestors/public-keys/add#--pkix-public-key-algorithm).
+- Cloud KMS using with `-kms_key_name` and `-kms_digest_alg`. 
```

Comment:
"Cloud KMS using with `-kms_key_name` and `-kms_digest_alg`." -> "Cloud KMS, using the `-kms_key_name` and `-kms_digest_alg` flags."

Reply:
done.